packages/core/stories/patterns/app-header/app-header.stories.tsx

```diff
@@ -0,0 +1,218 @@
+              }}
+            >
+              <FlexItem
+                style={{
+                  justifyContent: "center",
+                  display: "flex",
+                  height:
+                    "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+                  width:
+                    "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+                  borderRight:
+                    "var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-secondary-borderColor)",
+                }}
+              >
+                <Button
+                  onClick={handleDrawerToggle}
+                  variant={drawerOpen ? "primary" : "secondary"}
+                  style={{ alignSelf: "center" }}
+                >
+                  {drawerOpen ? <CloseIcon /> : <MenuIcon />}
+                </Button>
+              </FlexItem>
+              <FlexItem align="center">App logo</FlexItem>
+            </StackLayout>
+            <Drawer
+              style={{
+                paddingTop:
+                  "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+                paddingLeft: "0",
+              }}
+              open={drawerOpen}
+            >
+              <nav>
+                <ul style={{ listStyle: "none", padding: "0" }}>
+                  {items?.map((item) => (
+                    <li key={item}>
+                      <NavigationItem
+                        orientation="vertical"
+                        active={active === item}
+                        href="#"
+                        onClick={() => setActive(item)}
+                      >
+                        {item}
+                      </NavigationItem>
+                    </li>
+                  ))}
+                  {utilities?.map((utility) => (
+                    <li key={utility.key}>
+                      <NavigationItem orientation="vertical" href="#">
+                        {utility.icon}
+                        {utility.label}
+                      </NavigationItem>
+                    </li>
+                  ))}
+                </ul>
+              </nav>
+            </Drawer>
+          </>
+        ) : (
+          // Desktop Header
+          <FlexLayout
+            style={{
+              height: "calc(var(--salt-size-base) + var(--salt-spacing-200))",
+              paddingLeft: "1em",
+              paddingRight: "1em",
+              backgroundColor: "var(--salt-container-primary-background)",
+              position: "fixed",
+              inset: "0 0 auto 0",
+              borderBottom:
+                "var(--salt-size-border) var(--salt-container-borderStyle) var(--salt-container-secondary-borderColor)",
+            }}
+            justify="space-between"
+            gap={3}
+          >
+            <FlexItem align="center">App logo</FlexItem>
```

Comment:
The App logo and Main content text aren't using Salt font styles due to the UITK Salt compatibility changes

cc @origami-z

Reply:
using text component now